habitat/tasks/nav/nav.py

```diff
@@ -821,6 +824,14 @@ def _draw_shortest_path(
                 self.line_thickness,
             )
 
+    def _is_on_same_floor(self, pt, ref_floor_pt=None, ceiling_height=2.0):
```

Comment:
```suggestion
    def _is_on_same_floor(self, height, ref_floor_height=None, ceiling_height=2.0):
```

Since it is just the y component in the function call.  This looks good other than this!